src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -2372,103 +2190,24 @@ contract AllocationManagerUnitTests_ModifyAllocations is AllocationManagerUnitTe
-            _checkAllocationEvents({
-                operator: defaultOperator,
-                operatorSet: allocateParams[i].operatorSet,
-                strategy: strategyMock,
-                magnitude: allocateParams[i].newMagnitudes[0],
-                encumberedMagnitude: _encumberedMagnitudes[strategyMock] + allocateParams[i].newMagnitudes[0],
-                effectBlock: _defaultAllocEffectBlock()
-            });
-            _encumberedMagnitudes[strategyMock] += allocateParams[i].newMagnitudes[0];
-        }
-
-        cheats.prank(defaultOperator);
-        allocationManager.modifyAllocations(defaultOperator, allocateParams);
-
-        assertEq(
-            allocationManager.getAllocatableMagnitude(defaultOperator, strategyMock),
-            0,
-            "operator should not have any remaining allocatable magnitude"
-        );
 
-        // Move forward to allocation completion
+        AllocateParams[] memory allocateParams = _randAllocateParams_SingleMockStrategy_AllocAll(deallocSets);
+        _modifyAllocationsWithChecks(defaultOperator, allocateParams);
         cheats.roll(block.number + DEFAULT_OPERATOR_ALLOCATION_DELAY);
 
         // Deallocate fully from each operator set
         AllocateParams[] memory deallocateParams = _newAllocateParams(deallocSets, 0);
-
-        for (uint256 i; i < numOpSets; ++i) {
-            _checkDeallocationEvent({
-                operator: defaultOperator,
-                operatorSet: deallocSets[i],
-                strategy: strategyMock,
-                magnitude: 0,
-                effectBlock: uint32(block.number + DEALLOCATION_DELAY)
-            });
-        }
-
-        cheats.prank(defaultOperator);
-        allocationManager.modifyAllocations(defaultOperator, deallocateParams);
-
-        assertEq(
-            allocationManager.getAllocatableMagnitude(defaultOperator, strategyMock),
-            0,
-            "operator should still not have any allocatable magnitude"
-        );
-
-        // Move forward to deallocation completion
+        _modifyAllocationsWithChecks(defaultOperator, deallocateParams);
         cheats.roll(block.number + DEALLOCATION_DELAY);
 
-        // Check that we now have sufficient allocatable magnitude
-        assertEq(
-            allocationManager.getAllocatableMagnitude(defaultOperator, strategyMock),
-            WAD,
-            "operator should have all magnitude allocatable"
-        );
-
         // Create and register for a new operator set with the same default strategy.
         // If we try to allocate to this new set, it should clear the deallocation queue,
         // allowing all magnitude to be allocated
         OperatorSet memory finalOpSet = _createOperatorSet(OperatorSet(defaultAVS, r.Uint32()), defaultStrategies);
         _registerForOperatorSet(defaultOperator, finalOpSet);
-        AllocateParams[] memory finalAllocParams = _newAllocateParams(finalOpSet, WAD);
-
-        _checkClearDeallocationQueueEvents({
```

Comment:
Commenting this out is causing failures likely since deallocation events are different